os/kernel/binary_manager/binary_manager_verify.c

```diff
@@ -181,7 +182,7 @@ int binary_manager_read_header(int type, char *devpath, void *header_data, bool
 			read_size = bin_size < crc_bufsize ? bin_size : crc_bufsize;
 			ret = read(fd, (void *)crc_buffer, read_size);
 			if (ret < 0 || ret != read_size) {
-				bmdbg("Fail to read header for checksum : %d, errno %d\n", ret, errno);
+				bmdbg("Fail to read header for checksum : ret %d, errno %d\n", ret, errno);
```

Comment:
I think only 'ret != read_size' is enough, because ret is negative, then it is always different with read_size.

Reply:
Modified.